sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -0,0 +1,126 @@
+from six import python_2_unicode_compatible
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Any
+
+    from azure.core.credentials import TokenCredential
+    from azure.core.pipeline.transport import HttpRequest, HttpResponse
+
+from ._generated import AzureAttestationRestClient
+from ._generated.models import AttestationType, PolicyResult
+from ._configuration import AttestationClientConfiguration
+from ._models import AttestationSigner, AttestationToken, AttestationResponse, StoredAttestationPolicy, SigningKey
+from ._common import Base64Url
+import cryptography
+import cryptography.x509
+import base64
+from typing import List, Any
+from azure.core.tracing.decorator import distributed_trace
+from threading import Lock, Thread
+
+
+class AttestationAdministrationClient(object):
+    """Describes the interface for the per-tenant enclave service.
+    :param str base_url: base url of the service
+    :param credential: An object which can provide secrets for the attestation service
+    :type credential: azure.TokenCredentials or azure.AsyncTokenCredentials
+    :keyword Pipeline pipeline: If omitted, the standard pipeline is used.
+    :keyword HttpTransport transport: If omitted, the standard pipeline is used.
+    :keyword list[HTTPPolicy] policies: If omitted, the standard pipeline is used.
+    """
+
+    def __init__(
+        self,
+        credential,  # type: "TokenCredential"
+        instance_url,  # type: str
+        **kwargs  # type: Any
+    ):
+        # type: (str, Any, dict) -> None
+        base_url = '{instanceUrl}'
+        if not credential:
+            raise ValueError("Missing credential.")
+        self._config = AttestationClientConfiguration(credential, instance_url, **kwargs)
+        self._client = AzureAttestationRestClient(credential, instance_url, **kwargs)
+        self._statelock = Lock()
+        self._signing_certificates = None
+
+    @distributed_trace
+    def get_policy(self, attestation_type, **kwargs): 
+        #type(AttestationType) -> AttestationResult[str]:
+        """ Retrieves the attestation policy for a specified attestation type.
+        :param attestation_type - The attestation parameter type.
+        :type attestation_type: AttestationType
+        """
+        
+        policyResult = self._client.policy.get(attestation_type, **kwargs)
+        token = AttestationToken[PolicyResult](token=policyResult.token, body_type=PolicyResult)
+        token_body = token.get_body()
+        stored_policy = AttestationToken[StoredAttestationPolicy](token=token_body.policy, body_type=StoredAttestationPolicy)
+
+        actual_policy = stored_policy.get_body().attestation_policy #type: bytes
+
+        if self._config.token_validation_options.validate_token:
+            token.validate_token(self._config.token_validation_options, self._get_signers(**kwargs))
+
+        return AttestationResponse[str](token, actual_policy.decode('utf-8'))
```

Comment:
Interesting - FWIW, the CI pipeline doesn't complain about this, and I would have thought that this would be caught in the CI when it tested on 2.7. Or do we need to support versions of Python older than the versions in the CI pipeline?